pkg/mimedb/db.go

```diff
@@ -1524,6 +1612,10 @@ var DB = map[string]struct {
 		ContentType:  "video/jpm",
```

Comment:
is `jpm` supposed to be `image/jpm`

Reply:
its also video/jpm @kannappanr this file is usually just a guess work since its a map only unique keys are allowed, for correct one users have to provide themselves. It is not meant to be all correct all the time.  Because there is no one proper standard for Content-Types